source/extensions/filters/http/local_ratelimit/local_ratelimit.cc

```diff
@@ -130,16 +130,18 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
   config->stats().rate_limited_.inc();
 
   if (!config->enforced()) {
-    config->requestHeadersParser().evaluateHeaders(headers, decoder_callbacks_->streamInfo());
+    config->requestHeadersParser().evaluateHeaders(headers, headers,
+                                                   decoder_callbacks_->streamInfo());
```

Comment:
May be use the two parameters help function here?

Reply:
Yes. Done.